src/transformers/models/detr/image_processing_detr.py

```diff
@@ -1112,18 +1097,8 @@ def preprocess(
             )
             do_pad = kwargs.pop("pad_and_return_pixel_mask")
 
-        max_size = None
-        if "max_size" in kwargs:
-            warnings.warn(
-                "The `max_size` argument is deprecated and will be removed in a future version, use"
-                " `size['longest_edge']` instead.",
-                FutureWarning,
-            )
-            size = kwargs.pop("max_size")
-
         do_resize = self.do_resize if do_resize is None else do_resize
         size = self.size if size is None else size
-        size = get_size_dict(size=size, max_size=max_size, default_to_square=False)
```

Comment:
As above, please don't remove this line, and update to reflect the width, height ordering:
`size = get_size_dict(size=size, height_width_order=False, default_to_square=False)`